src/index.html

```diff
@@ -1,13 +1,319 @@
+          </article>
+        </div>
+      </section>
+
+      <section class="categories" id="categories">
+        <h2 class="section__title">
+          Browse Nothing
+          <br>
+          products by category
+        </h2>
+
+        <div class="categories__wrapper">
+          <article class="category">
+            <div class="category__photos">
+              <a href="https://www.apple.com/store" class="category__link category__link--wide">
+                <img
+                  class="category__photo"
+                  src="images/categories/all products - wide.png"
+                  alt="All products"
+                >
+              </a>
+
+              <a href="https://www.apple.com/store" class="category__link category__link--square">
+                <img
+                  class="category__photo"
+                  src="images/categories/all products - square.png"
+                  alt="All products"
+                >
+              </a>
+            </div>
+
+            <h3 class="category__title">All products</h3>
+          </article>
+
+          <article class="category">
+            <div class="category__photos">
+              <a href="https://www.apple.com/airpods/" class="category__link category__link--square">
+                <img
+                  class="category__photo"
+                  src="images/categories/audio - square.png"
+                  alt="Audio"
+                >
+              </a>
+
+              <a href="https://www.apple.com/airpods/" class="category__link category__link--wide">
+                <img
+                  class="category__photo"
+                  src="images/categories/audio - wide.png"
+                  alt="Audio"
+                >
+              </a>
+            </div>
+
+            <h3 class="category__title">Audio</h3>
+          </article>
+
+          <article class="category">
+            <div class="category__photos">
+              <a href="https://www.apple.com/shop/accessories/all" class="category__link category__link--wide">
+                <img
+                  class="category__photo"
+                  src="images/categories/accessories - wide.png"
+                  alt="Accessories"
+                >
+              </a>
+
+              <a href="https://www.apple.com/shop/accessories/all" class="category__link category__link--square">
+                <img
+                  class="category__photo"
+                  src="images/categories/accessories - square.png"
+                  alt="Accessories"
+                >
+              </a>
+            </div>
+
+            <h3 class="category__title">Accessories</h3>
+          </article>
+
+          <div class="link-wrapper">
+            <a href="https://www.apple.com/store" class="categories__link">View all</a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <div class="background-between-section"></div>
+
+    <contact class="main">
```

Comment:
as i know we don't have `contact` tag in html. I propose you to move your `about-us` section to the main tag and then replcae   `<section class="contacts"` to `<footer class="contacts"`